flytekit/extras/pod_templates/attach_shm.py

```diff
@@ -0,0 +1,19 @@
+from flytekit.core.pod_template import PodTemplate
+
+
+def attach_shm(name: str, size: str) -> PodTemplate:
```

Comment:
re thomas's comment, i was also thinking you were mutating a pod template.  if it's just generating a default pod template with an shm, should we rename the function?